pr_review_queue.py

```diff
@@ -430,15 +445,26 @@ def main():
             print("No pull requests found that match our criteria. Exiting.")
             sys.exit(0)
 
-        message = ("Good morning, image builders! :meow_wave:")
-        if needs_reviewer != []:
-            message += "\n\n:frog-derp: *We need a reviewer*\n  • " + "\n  • ".join(needs_reviewer)
-        if needs_changes != []:
-            message += "\n\n:changes_requested: *We need changes*\n  • " + "\n  • ".join(needs_changes)
-        if needs_review != []:
-            message += "\n\n:frog-flushed: *We need a review*\n  • " + "\n  • ".join(needs_review)
-        if needs_conflict_resolution != []:
-            message += "\n\n:expressionless-meow: *Update required*\n  • " +  "\n  • ".join(needs_conflict_resolution)
+        if slack_format:
+            message = ("Good morning, image builders! :meow_wave:")
+            if needs_reviewer != []:
+                message += "\n\n:frog-derp: *We need a reviewer*\n  • " + "\n  • ".join(needs_reviewer)
+            if needs_changes != []:
+                message += "\n\n:changes_requested: *We need changes*\n  • " + "\n  • ".join(needs_changes)
+            if needs_review != []:
+                message += "\n\n:frog-flushed: *We need a review*\n  • " + "\n  • ".join(needs_review)
+            if needs_conflict_resolution != []:
+                message += "\n\n:expressionless-meow: *Update required*\n  • " +  "\n  • ".join(needs_conflict_resolution)
+        else:
+            message = ("Good morning team!")
+            if needs_reviewer != []:
+                message += "\n\n**We need a reviewer**\n  * " + "\n  * ".join(needs_reviewer)
+            if needs_changes != []:
+                message += "\n\n**We need changes**\n  * " + "\n  * ".join(needs_changes)
+            if needs_review != []:
+                message += "\n\n**We need a review**\n  * " + "\n  * ".join(needs_review)
+            if needs_conflict_resolution != []:
+                message += "\n\n**Update required**\n  * " +  "\n  * ".join(needs_conflict_resolution)
```

Comment:
This indeed duplicates the code. So I could move the listing marker into variable, possibly even marker for bold text but I feel like icons would be too annoying same as `Good morning, image builders` :)

Reply:
How about we deduplicate this to "Good morning, everyone!" ?

Reply:
Right, but it does not help with everything other. Surely we don't want emojis all over the place :)